tools/purge_ems.rb

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env ruby
+require File.expand_path('../config/environment', __dir__)
+require 'optimist'
+
+ARGV.shift if ARGV[0] == '--' # if invoked with rails runner
+batch_message = "For each relationship, the number of rows to be removed in a batch. A lower number may decrease memory usage but also take longer."
+opts = Optimist.options do
+  banner "Purge management system records in the background via the queue.\n\nUsage: ruby #{$PROGRAM_NAME} [options]\n\nOptions:\n\t"
+  opt :id,      "Management System id",                                         :type => :integer, :required => true
+  opt :follow,  "Follow the progress or exit after creating work items.",       :type => :boolean, :default => true
+  opt :batch,   batch_message,                                                  :type => :integer, :default => 1_000
+  opt :timeout, "The number of minutes each work item will be allowed to run.", :type => :integer, :default => 30
```

Comment:
@agrare the cop was complaining because I indented `:required` on line 9 so the `:default` was lined vertically.